tox.ini

```diff
@@ -6,3 +6,9 @@ envlist = py35, p36
 commands = make check
 recreate = True
 whitelist_externals = make
+
+[flake8]
+# E731: Use a def instead of lambda expr
+ignore = E731
+application-package-names = xandikos
+max-line-length=100
```

Comment:
Please keep this at 80, per PEP8 (I realise there's quite a few place where we go over that at the moment).